themes/default/content/registry/packages/command/_index.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Command (preview)
+title: Command
 meta_desc: The Pulumi Command Provider enables you to execute commands and scripts either locally or remotely as part of the Pulumi resource model.
 layout: package
 ---
 
-The Pulumi Command Provider enables you to execute commands and scripts either locally or remotely as part of the Pulumi resource model.  Resources in the command package support running scripts on `create` and `destroy` operations, supporting stateful local and remote command execution.
+The Pulumi Command Provider enables you to execute commands and scripts either locally or remotely as part of the Pulumi resource model.  Resources in the command package support running scripts on `create` and `destroy` operations, supporting stateful local and remote command execution. In addition, the package provides a resource for copying assets to remote hosts via SSH.
```

Comment:
Nit. I would suggest placing this after the first sentence and maybe rephrasing as "The package also includes a resource for copying assets to remote hosts via SSH"